### ISSUE TITLE
Bug when drag cropping images after window resize and wrong original sizes format

### ISSUE THREAD

**Issue by thomaskanzig** [sulu/sulu#5372]
| Q | A
| --- | ---
| Sulu Version | 2.1
| Browser Version | Version 83.0.4103.116

# Actual Behavior

Here's a video to demonstrate how it's currently behaving when trying to drag after resize window:

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/1820440/86021196-211a0400-ba29-11ea-8e53-5bd46fdbdeaa.gif)

And here the original format of the image displayed wrong by the method [`$this->isInsideImage()`](https://github.com/sulu/sulu/blob/master/src/Sulu/Bundle/MediaBundle/Media/ImageConverter/Cropper/Cropper.php#L49)  

<img width="607" alt="Bildschirmfoto 2020-06-29 um 16 45 43" src="https://user-images.githubusercontent.com/1820440/86022480-bd90d600-ba2a-11ea-9953-9cdb80e48f3c.png">

# Expected Behavior

1. Possibility to drag to right in crop after window resize.
2. The original format size should display correctly in **$this->isInsideImage()**

# Steps to Reproduce

1. Upload the image.

![marvel-cinematic-universe-mcu-phase-4](https://user-images.githubusercontent.com/1820440/86028454-30517f80-ba32-11ea-8158-7d491a8042c2.jpg)

2. Create a new image format with a higher width/height than the initial image
```
    <format key="570x440">
        <meta>
            <title lang="de">Informationen</title>
            <title lang="en">Information</title>
        </meta>

        <scale x="570" y="440"/>
    </format>
```
3. Choose the image format when cropping
4. Observe

**Comment by nnatter** [comment]
Hey, 
can you check if #5399 fixes your problem?

**Comment by jordygroote** [comment]
I'm also experiencing problem cutting out images.
This is the format key:
```
<format key="image_element">
        <meta>
            <title lang="nl">Afbeelding element</title>
        </meta>

        <scale x="1280"/>
    </format>
```
Debugging says:
$image->getSize()->getWidth() = 4896

$width = 4899
$x = 0

```
if ($x + $width > $image->getSize()->getWidth()) {
    return false;
}
```
The width is larger than the image itself. So returns false…


Included is a printscreen with information and the source file (added red block for privacy reasons.):
![image](https://user-images.githubusercontent.com/8653367/87789529-7cc4f980-c83f-11ea-9e89-8d687eb9447c.png)
![dscf1432-1469197199-57922b8fd7beb](https://user-images.githubusercontent.com/8653367/87789391-438c8980-c83f-11ea-808d-5ddfa41105b5.jpeg)

**Comment by nnatter** [comment]
@jordygroote Thanks for the example! What is the size of your browser window when saving the crop?

**Comment by jordygroote** [comment]
Sorry forgot about that:
```
window.innerWidth = 1440
window.innerHeight = 798
```

13 " Macbook pro chrome browser.

**Comment by nnatter** [comment]
Thanks! I can reproduce the problem with your parameters. Unfortunately, it looks like #5399 does not fix it 😕

**Comment by jordygroote** [comment]
> Thanks! I can reproduce the problem with your parameters. Unfortunately, it looks like #5399 does not fix it 😕 We will need to have another look on this next week.

Happy that you can reproduce the issue and thanks for looking into it. Have a nice weekend!

**Comment by nnatter** [comment]
@jordygroote Can you try if #5422 fixes the issue for you too? 🙂

**Comment by jordygroote** [comment]
> @jordygroote Can you try if #5422 fixes the issue for you too? 🙂

I am with vacation for 3 weeks. Will ask collegue go check!

**Comment by nnatter** [comment]
Perfect - have a nice vacation! 🙂

**Comment by danrot** [comment]
Fixed in #5399
